### PR TITLE
updated the procedure to get more info for a barcode

### PR DESCRIPTION
**Pull request by RNAer**
this is for barcode scan utilities

### CODE REVIEW

**Comment by douginator2000** [comment]
Thanks! Suggest merging this one file with a larger pull requests for the
entire barcode scanning update.

On Wed, Jun 26, 2013 at 12:10 PM, Zech Xu notifications@github.com wrote:

> ## this is for barcode scan utilities
> 
> You can merge this Pull Request by running
> 
>   git pull https://github.com/RNAer/qiime_web_app master
> 
> Or view, comment on, or merge it at:
> 
>   https://github.com/qiime/qiime_web_app/pull/448
> Commit Summary
> - updated the procedure to get more info for a barcode
> 
> File Changes
> - _M_ database/procedures/ag_check_barcode_status.sqlhttps://github.com/qiime/qiime_web_app/pull/448/files#diff-0(13)
> 
> Patch Links:
> - https://github.com/qiime/qiime_web_app/pull/448.patch
> - https://github.com/qiime/qiime_web_app/pull/448.diff

## 

Doug Wendel
Professional Research Associate
University of Colorado at Boulder
Department of Chemistry and Biochemistry, Knight Lab
303-641-5442
wendel@colorado.edu

File: python_code/data_access/ag_data_access.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ def logParticipantSample(self, barcode, sample_site, environment_sampled, sample
     def deleteSample(self, barcode, ag_login_id):
         """
         Strictly speaking the ag_login_id isn't needed but it makes it really hard to hack
-        the function when you would need to know someone else's login id (a GUID) to
```
Comment (douginator2000) [inline python_code/data_access/ag_data_access.py:284]: Extra lines everywhere?

File: www/wwwadmin/barcode_util.psp [hunk 2]
```diff
@@ -1,6 +1,226 @@
+<%
+__author__ = "Zhenjiang (Zech) Xu"
```
Comment (douginator2000) [inline www/wwwadmin/barcode_util.psp:2]: Generally, do not replace the author with your name if the file was originally authored by someone else. Just add yourself to the credits and possibly the maintainer if you think you'll be the primary maintainer. But leave the original author.

File: www/wwwadmin/style/qiime.css [hunk 3]
```diff
@@ -1,164 +1,164 @@
 html, body {
```
Comment (douginator2000) [inline www/wwwadmin/style/qiime.css:1]: Making changes to CSS files - have you cleared this with Meg? Need to make absolutely sure of what you're changing as this touches the entire site (especially changing stuf fin the html and body tags).

File: www/wwwadmin/style/qiime.css [hunk 4]
```diff
@@ -1,164 +1,164 @@
...
+    opacity: 0.9;
+    padding: 8px;
+    border: 2px black solid;
+    border-radius: 5px;
+    font: 15px normal Arial, Helvetica, sans-serif;
+    width: 270px;
 }
 
 .login {
-	float:left;
```
Comment (douginator2000) [inline www/wwwadmin/style/qiime.css:49]: In general - whatever software you're using seem to be reformatting the entire file - different spacing and extra line breaks. Makes diffs really confusing and also generally don't need to reformat every file you open.

File: python_code/data_access/ag_data_access.py (hunk 1 above)
Comment (RNAer) [inline python_code/data_access/ag_data_access.py:284]: that's the whitespace cleaning I mentioned. sorry about it. I disabled it, but could we commit it in this time?

File: www/wwwadmin/barcode_util.psp (hunk 2 above)
Comment (RNAer) [inline www/wwwadmin/barcode_util.psp:2]: I mistakenly thought I made this file... I'll look it up who created it.

File: www/wwwadmin/style/qiime.css (hunk 4 above)
Comment (RNAer) [inline www/wwwadmin/style/qiime.css:49]: I'll correct this

**Pull request closed without merging**